src/rgw/rgw_op.h

```diff
@@ -1590,6 +1593,8 @@ class RGWHandler {
   virtual int authorize() = 0;
   virtual int postauth_init() = 0;
   virtual int error_handler(int err_no, string *error_content);
+  virtual bool set_rgw_err(int err_no, bool& is_website_redirect, int& http_ret, string& code) { return false;}
```

Comment:
`is_website_redirect` appears to be an input parameter, right? if that's the case, it shouldn't be a reference